src/runtime/audience-action-flow.js

```diff
@@ -163,6 +164,8 @@ export class AudienceActionFlow {
     } else {
       this.showFailedOptedInToast_();
     }
+    const now = Date.now().toString();
+    this.deps_.storage().set(Constants.READ_TIME, now, false);
```

Comment:
`useLocalStorage` defaults to false so you shouldn't need to provide the explicit value. If you want to, maybe include a comment to indicate what the param name is
```js
this.deps_.storage().set(Constants.READ_TIME, now, /*useLocalStorage*/ false);
```

Reply:
Done. Opted to keep the param with a comment because I remember being confused by this in the past.